CONTRIBUTING.md

```diff
@@ -16,62 +17,83 @@ First of all, thanks for taking the time to contribute! This project can only gr
 </ul>
 
 <hr>
-<h2 id="overview">Overview on Submitting Icon</h2>
+<h2 id="terms">Terms</h2>
+<p>Here are some terms that we will use in this repo: </p>
+<ol>
+  <li>"Icon" refers to the set of svgs/icons of a technology/tool. Ex: We might refer to the React svgs and React icons as the React Icon</li>
+  <li>"SVG/<code>.svg</code>" refers to the <code>svg</code> versions of the Icons.</li>
+  <li>"icons" refers to the icon versions of the Icons.</li>
```

Comment:
Confusing sentence, what do you want to express? But I like the idea of **Terms**

Reply:
I notice that we use "icon" interchangeably for both the technology/tool and the icons of those tools in our repo. So I wanted to define the "Icon" capitalize refers to the tech/tool while the "icon" lower case just means a font icon. I can try to improve the definition for sure.